standard/expressions.md

```diff
@@ -890,6 +897,7 @@ The ***inferred return type*** is determined as follows:
 >
 > The following example demonstrates how anonymous function type inference allows type information to “flow” between arguments in a generic method invocation. Given the method:
 >
+> <!-- Example: {template:"standalone-lib", name:"InferredReturnType2", replaceEllipsis:true, expectedOutput:["x", "x", "x"], expectedErrors:["x","x"], expectedWarnings:["x","x"]} -->
```

Comment:
And again - I won't note further ones, but basically I'd like the shell script to pass on your machine before we merge.

I'd also note that if we're expecting output, standalone-lib is the wrong template :)

Reply:
agreed